Add Catch the Beat Indonesia Cup 2020 (#3946)
ppy/osu-wiki: wiki/Tournaments/CIC/2020/en.md

---
tags:
  - CIC2020 
  - CIC 2020
---

# Catch the Beat Indonesia Cup 2020

> ![CIC 2020 banner](img/logo.jpg)

Reviewer: ```suggestion
![CIC 2020 logo](img/logo.jpg)
```

> The **Catch the Beat Indonesia Cup 2020** (***CIC 2020***) is a double-elimination 1v1 osu!catch tournament hosted by ![][flag_ID] [xHirayuki](https://osu.ppy.sh/users/3245206), ![][flag_ID] [CamXDanna](https://osu.ppy.sh/users/3243148), ![][flag_ID] [Eum](https://osu.ppy.sh/users/16552751), and ![][flag_ID] [Kazuyo](https://osu.ppy.sh/users/5604201). The tournament is open to all osu!catch players from Indonesia with over 3000 playcounts regardless of rank. It is the eighth iteration of the tournament series.

Reviewer: ```suggestion
The **Catch the Beat Indonesia Cup 2020** (***CIC 2020***) is a double-elimination 1v1 osu!catch tournament hosted by ![][flag_ID] [xHirayuki](https://osu.ppy.sh/users/3245206), ![][flag_ID] [CamXDanna](https://osu.ppy.sh/users/3243148), ![][flag_ID] [Eum](https://osu.ppy.sh/users/16552751), and ![][flag_ID] [Kazuyo](https://osu.ppy.sh/users/5604201). The tournament is open to all osu!catch players from Indonesia with over 3,000 playcounts regardless of rank. It is the eighth iteration of the tournament series.
```

> The **Catch the Beat Indonesia Cup 2020** (***CIC 2020***) is a double-elimination 1v1 osu!catch tournament hosted by ![][flag_ID] [xHirayuki](https://osu.ppy.sh/users/3245206), ![][flag_ID] [CamXDanna](https://osu.ppy.sh/users/3243148), ![][flag_ID] [Eum](https://osu.ppy.sh/users/16552751), and ![][flag_ID] [Kazuyo](https://osu.ppy.sh/users/5604201). The tournament is open to all osu!catch players from Indonesia with over 3,000 osu!catch playcounts regardless of rank. It is the eighth iteration of the tournament series.

Reviewer 2: ```suggestion
The **Catch the Beat Indonesia Cup 2020** (***CIC 2020***) is a double-elimination 1v1 osu!catch tournament hosted by ![][flag_ID] [xHirayuki](https://osu.ppy.sh/users/3245206), ![][flag_ID] [CamXDanna](https://osu.ppy.sh/users/3243148), ![][flag_ID] [Eum](https://osu.ppy.sh/users/16552751), and ![][flag_ID] [Kazuyo](https://osu.ppy.sh/users/5604201). The tournament is open to all osu!catch players from Indonesia with over 3,000 osu!catch playcount regardless of rank. It is the eighth installment of the Catch the Beat Indonesia Cup.
```

## Tournament schedule

| Event | Timestamp |
| --: | :-- |

> | Registration Phase | 2020-07-10/2020-07-20 |
> | Screening Phase | 2020-07-21/2020-07-27 |

Reviewer 2: ```suggestion
| Registration phase | 2020-07-10/2020-07-20 |
| Screening phase | 2020-07-21/2020-07-27 |
```

> | Group stage | 2020-07-28/2020-08-02 |

Reviewer 2: ```suggestion
| Group Stage | 2020-07-28/2020-08-02 |
```

| Round of 16 | 2020-08-03/2020-08-09 |
| Quarterfinals | 2020-08-10/2020-08-16 |
| Semifinals | 2020-08-17/2020-08-23 |

> | Finals week 1 | 2020-08-24/2020-08-30 |
> | Finals week 2 | 2020-08-31/2020-09-06 |

Reviewer 2: ```suggestion
| Finals - Week 1 | 2020-08-24/2020-08-30 |
| Finals - Week 2 | 2020-08-31/2020-09-06 |
```

## Prizes

| Placing | Prize(s) |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 50% of the raised prize pool, 3 months of osu!supporter tag, unique profile badge |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 30% of the raised prize pool, 2 months of osu!supporter tag |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 20% of the raised prize pool, 1 month of osu!supporter tag |
| 4th place | 1 month of osu!supporter tag |

## Organization

> OSIM is run by various osu! community members predominantly hailing from Indonesia.

Reviewer: ```suggestion
CIC 2020 is run by various osu! community members predominantly hailing from Indonesia.
```

> | Position | Members |

Reviewer: ```suggestion
| Position | Member(s) |
```
Some position only taken by one person

| Referee | ![][flag_ID] [xHirayuki](https://osu.ppy.sh/users/3245206), ![][flag_ID] [-Hex-](https://osu.ppy.sh/users/8630988), ![][flag_ID] [mina-](https://osu.ppy.sh/users/4670692), ![][flag_ID] [Avoya](https://osu.ppy.sh/users/3136817), ![][flag_ID] [Niva](https://osu.ppy.sh/users/197805), ![][flag_ID] [Yuina Hotaru](https://osu.ppy.sh/users/1104256),  ![][flag_ID] [- Ruu -](https://osu.ppy.sh/users/3811821), ![][flag_ID] [Koimeji](https://osu.ppy.sh/users/4137039), ![][flag_ID] [Execration-](https://osu.ppy.sh/users/9260926), ![][flag_ID] [Valskiel](https://osu.ppy.sh/users/1254639) |
| Graphic designer | ![][flag_ID] [mina-](https://osu.ppy.sh/users/4670692), ![][flag_ID] [Avoya](https://osu.ppy.sh/users/3136817) |
| Spreadsheet manager | ![][flag_ID] [Kazuyo](https://osu.ppy.sh/users/5604201) |
| Wiki editor | ![][flag_ID] [Niva](https://osu.ppy.sh/users/197805) |

## Links

- **[Main spreadsheet](http://docs.google.com/spreadsheets/u/3/d/e/2PACX-1vRWcBzTJy5RnGqGIzRDc_Zv28lRo9Z3yz9yUVqF4eqoVi-cebrHDDEWk186ODj71mLf2x9mPARwaVlB/pubhtml?gid=2045538686&single=true)**

> - [Forum thread](https://osu.ppy.sh/community/forums/topics/1096883)

Reviewer 2: ```suggestion
- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1096883)
```

- [Discord server](https://discord.gg/YwAYbPa)

> - [Livestream channel](https://www.twitch.tv/osuIndonesia/)

Reviewer 2: ```suggestion
- [Livestream](https://www.twitch.tv/osuIndonesia/)
```

> ## Participants
> 
> | Group |  |  |  |  |
> | :-: | :-- | :-- | :-- | :-- |
> | **A** | TBD | TBD | TBD | TBD |
> | **B** | TBD | TBD | TBD | TBD |
> | **C** | TBD | TBD | TBD | TBD |
> | **D** | TBD | TBD | TBD | TBD |
> | **E** | TBD | TBD | TBD | TBD |
> | **F** | TBD | TBD | TBD | TBD |
> | **G** | TBD | TBD | TBD | TBD |
> | **H** | TBD | TBD | TBD | TBD |
> 

Reviewer 2: i guess if you are not planning to fill the table in this PR, its better remove the entire header for now 
```suggestion
```

Author: i can update it in a bit when the groups are announced tonight i think, the reason i had a template in place was because the screening result hasn't came in yet at the time of writing

Author: added participants, all should be good now

## Mappools

> ### Group stage

Reviewer 2: ```suggestion
### Group Stage
```

**[Download the mappack here! (111 MB)](https://drive.google.com/file/d/184os6Qkk7YKnSUVB_DP59PJ7mELzZo4N/view?usp=sharing)**

- NoMod

>   - [Ryu* - Sakura Mirage (Lacrima) \[EXHAUST\]](https://osu.ppy.sh/beatmapsets/1051862#fruits/2201873)

Reviewer 2: ```suggestion
  - [Ryu\* - Sakura Mirage (Lacrima) \[EXHAUST\]](https://osu.ppy.sh/beatmapsets/1051862#fruits/2201873)
```

> | **H** | ![][flag_ID] [Moncar321](https://osu.ppy.sh/users/1400073) | ![][flag_ID] [Toka](https://osu.ppy.sh/users/1595221) | ![][flag_ID] [Mmomchuuan21](https://osu.ppy.sh/users/8794848) | ![][flag_ID] [Azwa_Kurendra](https://osu.ppy.sh/users/14799356) |

Reviewer 2: ```suggestion
| **H** | ![][flag_ID] [Moncar321](https://osu.ppy.sh/users/1400073) | ![][flag_ID] [Toka](https://osu.ppy.sh/users/1595221) | ![][flag_ID] [Mmomchuuan21](https://osu.ppy.sh/users/8794848) | ![][flag_ID] [Azwa\_Kurendra](https://osu.ppy.sh/users/14799356) |
```
